### PR TITLE
Rename SKUs to help Autorest generating a good name

### PR DESCRIPTION
**Pull request by lmazuel**
Fix bad naming with Autorest:
- SKUs gives sk_us_operations
- Skus gives skus_operations

### CODE REVIEW

File: specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
         "tags": [
           "SKU"
```
Comment (amarzavery) [inline specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json]: Skus

File: specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json [hunk 2]
```diff
@@ -53,7 +53,7 @@
...
         ],
-        "operationId": "SKUs_List",
+        "operationId": "Skus_List",
         "description": "Lists the available SKUs supported by Microsoft.Storage for given subscription.",
         "x-ms-examples": {
           "SKUList": { "$ref": "./examples/SKUList.json" }
```
Comment (amarzavery) [inline specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json]: SkuList.

To be consistent.

Comment (lmazuel) [reply specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json]: I renamed the `tags` to `Skus` is this enough? I would prefer the `s`, for consistency with `StorageAccounts`

**Review by JasonYang-MSFT — APPROVED** [review]

**Review by amarzavery — APPROVED** [review]

**Comment by amarzavery** [comment]
@JasonYang-MSFT - The PR has been merged. Please  regenerate the node sdk for storage management and update the existing PR.

**Pull request merged**